Source/Tools/relive_api/AETlvs.hpp

```diff
@@ -256,9 +258,9 @@ namespace AETlvs
 
         CTOR_AE(Path_ElectricWall, "ElectricWall", TlvTypes::ElectricWall_38)
         {
-            ADD("Scale", mTlv.field_10_scale);
-            ADD("ID", mTlv.field_12_id);
-            ADD("Start State", mTlv.field_14_start_state);
+            TLVOBJECTBASE_ADD("Scale", mTlv.field_10_scale);
```

Comment:
does this have to be renamed? its super verbose now and it hurts the readability somewhat

Reply:
Doesn't have to, but I think it's an improvement, `ADD` is too short of a name for a macro definition. Could easily collide with new macros in the future or if we add a new 3rd party dependency. Also makes it clear where the macro is defined.

Reply:
I think it should be left as ADD for now - could just add a comment in each of those AO/AE headers documenting where it comes from given its used in like 90% of the lines that make up these 2 files